From 4f13bbc7e326dc0e96db6c80f1fa86b8f6ef237b Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Mon, 30 Sep 2019 13:28:30 -0500
Subject: [PATCH] trivial: motd: disable updating motd for now

It looks like a systemd bug is happening, but let it be diagnosed upstream
systemd first.

https://github.com/systemd/systemd/issues/13688
---
 data/motd/fwupd-refresh.service.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/data/motd/fwupd-refresh.service.in b/data/motd/fwupd-refresh.service.in
index 2cfb131..aa5d77d 100644
--- a/data/motd/fwupd-refresh.service.in
+++ b/data/motd/fwupd-refresh.service.in
@@ -10,7 +10,9 @@ CacheDirectory=fwupdmgr
 RuntimeDirectoryPreserve=yes
 StandardError=null
 ExecStart=@bindir@/fwupdmgr refresh
-ExecStart=@bindir@/fwupdmgr get-updates --log @motd_file@
+#Don't update MOTD for now until https://github.com/systemd/systemd/issues/13688
+#is better figured out
+#ExecStart=@bindir@/fwupdmgr get-updates --log @motd_file@
 DynamicUser=yes
 RestrictAddressFamilies=AF_NETLINK AF_UNIX AF_INET AF_INET6
 SystemCallFilter=~@mount
-- 
2.7.4

